src/locale/locales/de/messages.po

```diff
@@ -2382,105 +2302,103 @@ msgstr "Ja"
 
 #: src/view/com/auth/create/Step1.tsx:106
 msgid "You can change hosting providers at any time."
-msgstr "Sie können den Hosting-Anbieter jederzeit wechseln."
+msgstr "Du kannst den Hosting-Anbieter jederzeit wechseln."
 
 #: src/view/com/auth/login/Login.tsx:158
 #: src/view/com/auth/login/PasswordUpdatedForm.tsx:31
 msgid "You can now sign in with your new password."
-msgstr "Sie können sich jetzt mit Ihrem neuen Passwort anmelden."
+msgstr "Du kannst dich jetzt mit deinem neuen Passwort anmelden."
 
 #: src/view/com/modals/InviteCodes.tsx:64
 msgid "You don't have any invite codes yet! We'll send you some when you've been on Bluesky for a little longer."
 msgstr "Du hast noch keine Einladungscodes! Wir schicken dir welche, wenn du schon etwas länger bei Bluesky bist."
 
 #: src/view/screens/SavedFeeds.tsx:102
 msgid "You don't have any pinned feeds."
-msgstr "Sie haben keine angehefteten Feeds."
+msgstr "Du hast keine angehefteten Feeds."
 
 #: src/view/screens/Feeds.tsx:383
 msgid "You don't have any saved feeds!"
-msgstr "Sie haben keine gespeicherte Feeds!"
+msgstr "Du hast keine gespeicherten Feeds!"
 
 #: src/view/screens/SavedFeeds.tsx:135
 msgid "You don't have any saved feeds."
-msgstr "Sie haben keine gespeicherte Feeds."
+msgstr "Du hast keine gespeicherten Feeds."
 
 #: src/view/com/post-thread/PostThread.tsx:381
 msgid "You have blocked the author or you have been blocked by the author."
-msgstr "Sie haben den Verfasser blockiert oder Sie wurden vom Verfasser blockiert."
+msgstr "Du hast den Verfasser blockiert oder du wurdest vom Verfasser blockiert."
 
 #: src/view/com/feeds/ProfileFeedgens.tsx:134
 msgid "You have no feeds."
-msgstr "Sie haben keine Feeds."
+msgstr "Du hast keine Feeds."
 
-#: src/view/com/lists/MyLists.tsx:89
-#: src/view/com/lists/ProfileLists.tsx:138
+#: src/view/com/lists/MyLists.tsx:89 src/view/com/lists/ProfileLists.tsx:138
 msgid "You have no lists."
-msgstr "Sie haben keine Listen."
+msgstr "Du hast keine Listen."
 
 #: src/view/screens/ModerationBlockedAccounts.tsx:132
 msgid "You have not blocked any accounts yet. To block an account, go to their profile and selected \"Block account\" from the menu on their account."
-msgstr "Sie haben noch keine Konten gesperrt. Um ein Konto zu sperren, gehen Sie zu seinem Profil und wählen Sie \"Konto sperren\" aus dem Menü auf seinem Konto."
+msgstr ""
 
 #: src/view/screens/AppPasswords.tsx:86
 msgid "You have not created any app passwords yet. You can create one by pressing the button below."
-msgstr "Sie haben noch kein Kennwort für eine Anwendung erstellt. Sie können eines erstellen, indem Sie auf die Schaltfläche unten drücken."
+msgstr "Du hast noch keine App-Passwörter erstellt. Du kannst eines erstellen, indem du auf die Schaltfläche unten klickst."
```

Comment:
`"Du hast noch keine App-Passwörter erstellt. Du kannst welche erstellen, indem du auf die Schaltfläche unten klickst."`
(now both are in plural)